.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 
 - [ ] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
 
-- [ ] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stack-structure.md).
+- [ ] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/lifecycle.md).
```

Comment:
I think this link might be more appropriate: https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure since it contains the structure of the stack.

Reply:
Yeah I think you're right... rebased and updated.